src/reference-manual/types.qmd

```diff
@@ -472,6 +472,27 @@ model {
 }
 ```
 
+For a container variable, the affine transform is applied to each element of that variable.
+As an example, the non-centered parameterization of Neal's Funnel in the
+[Stan User's Guide reparameterization section](https://mc-stan.org/docs/stan-users-guide/reparameterization.html),
+$$
+p(y,x) = \textsf{normal}(y \mid 0,3) \times \prod_{n=1}^9
+\textsf{normal}(x_n \mid 0,\exp(y/2)).
+$$
+can be written as:
```

Comment:
If you're going to treat the Stan program as a noun syntactically you don't need the colon.  I would suggest as an alternative to just say "can be written as follows and then on .ine 494 start a new sentence that say something like "In this program, the affine transform is applied to every element of vector `x`."